.github/workflows/push.yml

```
@@ -49,30 +50,32 @@ jobs:
           name: "${{env.CODECOV_UNIQUE_NAME}}"
-      - name: Upload test results to Codecov
-        if: ${{ !cancelled() }}
-        uses: codecov/test-results-action@v1
+      - name: Test acceptable level of code coverage
+        uses: VeryGoodOpenSource/very_good_coverage@v3
         with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-
+          path: ./coverage.lcov
+          min_coverage: 0.0
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)